website/server/controllers/api-v3/chat.js

```diff
@@ -300,6 +291,41 @@ api.clearChatFlags = {
       {$set: {'chat.$.flagCount': message.flagCount}}
     );
 
+    let adminEmailContent = getUserInfo(user, ['email']).email;
+
+    let author = await User.findOne({_id: message.uuid}, {auth: 1});
+
+    let authorEmailContent = getUserInfo(author, ['email']).email;
+
+    let groupUrl;
+    if (group._id === TAVERN_ID) {
```

Comment:
This logic is used in the flag route too, right? This should be abstracted out into a function that both can use then.